### PR TITLE
added band member area

### PR DESCRIPTION
**Pull request by Cucurrucho**

### CODE REVIEW

File: app/Http/Requests/Admin/BandMember/UpdateBandMemberRequest.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		$this->bandMember = $this->route('bandMember');
+		return $this->user()->can('update', $this->bandMember);
+	}
+
+	public function rules() {
+		return [
+			'name' => 'required',
+			'email' => 'required|email|unique:users,email,' . $this->bandMember->user->id,
+			'language' => 'required|in:en,nl',
+			'bandmember' => 'required|array'
```
Comment (LCD344) [inline app/Http/Requests/Admin/BandMember/UpdateBandMemberRequest.php]: this needs fields validation (required fields need be required)

File: app/Http/Requests/BandMember/UpdateBandMemberRequest.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	 *
+	 * @return array
+	 */
+	public function rules() {
+		return [
+			'name' => 'required',
+			'email' => 'required|email|unique:users,email,' . $this->bandMember->user->id,
+			'language' => 'required|in:en,nl',
+			'bandmember' => 'required|array'
+		];
```
Comment (LCD344) [inline app/Http/Requests/BandMember/UpdateBandMemberRequest.php]: This needs required fields validation

Comment (LCD344) [reply app/Http/Requests/BandMember/UpdateBandMemberRequest.php]: Also, it needs save and save and submit features

File: app/Http/Requests/Admin/BandMember/UpdateBandMemberRequest.php (hunk 1 above)
Comment (LCD344) [reply app/Http/Requests/Admin/BandMember/UpdateBandMemberRequest.php]: Also, it needs save and save and submit features

Comment (Cucurrucho) [reply app/Http/Requests/Admin/BandMember/UpdateBandMemberRequest.php]: what does the save and submit mean??

File: app/Http/Requests/BandMember/UpdateBandMemberRequest.php (hunk 2 above)
Comment (Cucurrucho) [reply app/Http/Requests/BandMember/UpdateBandMemberRequest.php]: same

**Pull request merged**